libreyolo/postprocess/rfdetr.py

```
@@ -9,14 +9,140 @@
 
 import torch
 import torch.nn.functional as F
-from typing import List, Dict
+from typing import List, Dict, Optional, Sequence
```

Ruff (0.15.17)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)